include/onnxruntime/core/framework/op_kernel.h

```diff
@@ -103,6 +104,8 @@ class OpKernelContext {
   // Return nullptr if the output is an unused optional output.
   Tensor* Output(int index, const TensorShape& shape);
 
+  SparseTensor* Output(int index, size_t num_values, const TensorShape& shape);
+
```

Comment:
As this is a public method in a public header it should probably have some documentation.

Reply:
Done